Source/WebCore/platform/cocoa/RuntimeApplicationChecksCocoa.mm

```diff
@@ -91,6 +91,22 @@ void clearApplicationBundleIdentifierTestingOverride()
 #endif
 }
 
+static String& presentingApplicationBundleIdentifierStorage()
+{
+    static NeverDestroyed<String> identifier;
```

Comment:
We likely want MainThreadNeverDestroyed here given that this doesn't look thread safe.

Reply:
I'll also add a `ASSERT(RunLoop::isMain())`, to verify this is only being accessed on the main thread.

Reply:
Aha, never mind. It looks like MainThreadNeverDestroyed already does that.

Reply:
Right, that is the whole point of MainThreadNeverDestroyed AFAIK :)